mlflow/server/js/src/experiment-tracking/components/ExperimentListView.js

```diff
@@ -207,49 +274,108 @@ export class ExperimentListView extends Component {
           onChange={this.handleSearchInputChange}
           data-test-id='search-experiment-input'
         />
-        <div css={classNames.experimentListContainer}>
-          <Tree
-            treeData={treeData}
-            dangerouslySetAntdProps={{
-              selectable: true,
-              checkable: true,
-              multiple: true,
-              selectedKeys: activeExperimentIds,
-              checkedKeys: activeExperimentIds,
-              onCheck: this.handleCheck,
-              titleRender: this.renderListItem,
-            }}
-          />
+        <div>
+          <AutoSizer>
+            {({ width, height }) => (
+              <VList
+                rowRenderer={this.renderListItem}
+                data={filteredExperiments}
+                ref={this.bindListRef}
+                rowHeight={32}
+                overscanRowCount={10}
+                height={height}
+                width={width}
+                rowCount={filteredExperiments.length}
+              />
+            )}
+          </AutoSizer>
         </div>
       </div>
     );
   }
 }
 
 const classNames = {
-  experimentListOuterContainer: {
+  experimentListOuterContainer: css({
     boxSizing: 'border-box',
+    height: '100%',
+    marginTop: '24px',
```

Comment:
let's remove this margin - after merging in UI changes with chart view, it causes an unnecessary scrollbar to appear:

https://user-images.githubusercontent.com/104438646/221513359-2a314b7a-5569-4155-b41d-77a8cc194a01.mov

```suggestion
```